src/algorithms.jl

```diff
@@ -22,6 +22,35 @@ Base.@pure RKMil(;interpretation=:Ito) = RKMil{interpretation}()
 struct RKMilCommute{interpretation} <: StochasticDiffEqAlgorithm end
 Base.@pure RKMilCommute(;interpretation=:Ito) = RKMilCommute{interpretation}()
 
+###############################################################################
+
+# Predictor Corrector
+struct PCEuler{T<:Real} <: StochasticDiffEqAlgorithm
+  theta::T
+  eta::T
+  bbprime::Function
```

Comment:
Parameterize this for stability purposes.

Reply:
Could you elaborate on this a little more? Would you like the PCEuler to be parametrized by the  type of the function bbprime?

Reply:
In Julia, every function is a different type and `Function` is just an abstract type. This allows specialization on functions and then allows things like inlining. If you `::Function` in a type, then it just knows it's a `<:Function` and cannot optimize directly on it. So yup, just make that `::F` and a free-roaming `F` parameter.

Reply:
I see, will make the change.